src/components/form/Obs.jsx

```diff
@@ -59,13 +61,14 @@ const Obs = (props) => {
           checkBoxValue={props.conceptAnswer}
           component={CheckBox}
           name={props.name}
+          onBlur={e => { e.preventDefault(); }}
```

Comment:
I added this to fix a know bug with redux-form resetting the value of check-box to booleans (https://github.com/erikras/redux-form/issues/2768)